Makefile

```diff
@@ -70,6 +71,9 @@ testclean:
 testfclean: testclean
 	$(PREFIX)rm -f $(TEST_FNAME)
 
+test_libft_fclean:
```

Comment:
Let's keep uniform naming. Either this can be "pretestfclean", or rename all the others from "pre" to "libft". I prefer pre, since in theory we also can have other dependencies (like minilibx)